db/re/skill_nocast_db.txt

```diff
@@ -5,7 +5,7 @@
 // SkillID,Flag
 //
 // Legend for 'Flag' field (bitmask):
-// 1    - cannot be used in normal maps
+// 1    - cannot be used in normal maps (enabled on pvp, gvg, gvg_dungeon, gvg_castle, gvg_te, gvg_te_castle, battleground)
```

Comment:
If we adjust the comment here, we should probably make the change in the pre-re and import files.